### PR TITLE
align parameters with executed values

### PR DESCRIPTION
**Pull request by 0xldr**
Aligns values in the Stability Scope with values that have been executed onchain through executive votes.

### CODE REVIEW

File: MIP104/MIP104.md [hunk 1]
```diff
@@ -883,7 +883,7 @@ Current WBTC-A parameters are:
 
 Current WBTC-B parameters are:
 
-- Stability Fee: 15.25%
+- Stability Fee: 10.30%
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
- Stability Fee: 10.25%
```

**Review by blimpa — CHANGES_REQUESTED** [review]

File: MIP104/MIP104.md (hunk 1 above)
Comment (0xldr) [reply MIP104/MIP104.md]: image

image

Good catch, Makerburn was rounding 10.25 - see image. We will report this to BA Labs as the rounding seems to be inconsistent - as 10.75 is not rounded to 10.8 on the same page.

File: MIP104/MIP104.md [hunk 2]
```diff
@@ -339,7 +339,7 @@ The Stability Facilitators can trigger executive votes that set up DDMs to Morph
 
 ¤¤¤
 
-Current target exposure of Morpho sUSDe or USDe vault is: 0
+Current target exposure of Morpho sUSDe or USDe vault is: 450,000,000 DAI
```
Comment (blimpa) [inline MIP104/MIP104.md:342]: I don't know where this value comes from, so I couldn't verify

**Review by blimpa — APPROVED** [review]
LGTM (otherwise)

File: MIP104/MIP104.md (hunk 2 above)
Comment (0xldr) [reply MIP104/MIP104.md:342]: https://forum.makerdao.com/t/spark-metamorpho-dai-vault-allocation/23987/18

"Vault target debt exposure: 450 million DAI"

**Pull request merged**